docs/pages/combobox/index.mdx

```diff
@@ -1,26 +1,37 @@
-# Combobox
-
+# ComboBox
 
+An accessible ComboBox with single, multi & accessible select with typeahead &
+autocomplete support.
```

Comment:
```suggestion
An accessible ComboBox Component with 
1. single
2. multi selectable 

with typeahead & autocomplete support. (Accessible in future)
```

Reply:
Docs changes, I did not spend much time in writing the docs now. These changes are fine now. We can improve it later.